oml/lightning/callbacks/metric.py

```diff
@@ -72,6 +89,30 @@ def on_validation_batch_end(
             if is_last_expected_batch:
                 self.calc_and_log_metrics(pl_module)
 
+    def _log_images(self, pl_module: pl.LightningDataModule) -> None:
+        if not isinstance(self.metric, EmbeddingMetrics):
```

Comment:
Since I suggested introducing a new interface we want to check if our code after that is really generalizable. We can make `TripletAccuracy` also a child of IMetricWithVisualization and add the test that our callback works for both types of metrics. 

* For simplicity in AccuracyOnTriplets we can only return a figure with a black square for now.